src/main.rs

```diff
@@ -220,32 +238,85 @@ impl FuzzProject {
            .arg("run")
            .arg("--manifest-path")
            .arg(self.manifest_path());
-        if release {
+        if !args.is_present("no_opt") {
             cmd.arg("--release");
         }
         cmd.arg("--verbose")
            .arg("--bin")
            .arg(&target)
            .arg("--target")
-           // won't pass rustflags to build scripts
-           .arg(target_triple)
+           .arg(utils::default_target()) // won't pass rustflags to build scripts
```

Comment:
Kind of -- this just uses the default triple target (`x86_64-apple-darwin` or `x86_64-unknown-linux-gnu`) instead of letting the user pass a triple target in.

The thing is, there's actually no way to pass in triple target right now anyways 😸 because the corresponding flag doesn't have a `takes_value(true)` (see https://github.com/rust-fuzz/cargo-fuzz/blob/master/src/main.rs#L57)!

Seeing as no one's run into this bug in the last month, I kind of figured that no one actually needed to override the default target and decided to just remove that functionality (especially becuase you can just set the `RUSTFLAGS` environment variable if you really need to). Of course, if you think it's important that we support this, I can also just add the `takes_value(true)` and fix the underlying bug.